specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-10-01-preview/RoleManagementPolicy.json

```diff
@@ -117,8 +117,8 @@
           }
         ],
         "responses": {
-          "201": {
-            "description": "Created - Returns the created or updated policy.",
+          "200": {
```

Comment:
why are you changing existing response codes? first time create should return a 201 and update on the same resource should return a 200 OK

Reply:
These resources are created in the system by default, so the user never does a PUT. If they want to change any exisiting properties, they can do PATCH to update.